microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md

```diff
@@ -59,38 +59,38 @@ To distribute your extension using the Windows registry:
    > [!NOTE]
    > If you want to install an extension from the Chrome Web Store, set the value of `update_url` to `https://clients2.google.com/service/update2/crx`.
 
-1. Verify that your extension is listed in Microsoft Edge, by going to `edge://extensions`.
+1. In Microsoft Edge, go to `edge://extensions` and verify that your extension is listed.
 
 
 <!-- ====================================================================== -->
 ## Use a preferences JSON file (macOS and Linux)
 
-To distribute your extension using a preferences JSON file:
+To distribute your extension by using a preferences JSON file:
 
-1. When using Linux, ensure your `.crx` extension file is available on the machine that the extension will be installed on. Copy the `.crx` extension file to a local directory, or use a  network share that is reachable from the machine.
+1. When using Linux, make sure your `.crx` extension file is available on the machine that the extension will be installed on.  Copy the `.crx` extension file to a local directory, or use a network share that is reachable from the machine.
 
-1. Create a JSON file where the name of the file corresponds to the ID of your extension. For example, create a JSON file with the file name `aaaaaaaaaabbbbbbbbbbcccccccccc.json`.
+1. Create a <!--todo: preferences?--> JSON file where the name of the file corresponds to the ID of your extension. For example, create a JSON file with the file name `aaaaaaaaaabbbbbbbbbbcccccccccc.json`.
 
-1. Depending on your operating system, save the JSON file to one of the following folders:
+1. Depending on your operating system, save the <!--todo: preferences?--> JSON file to one of the following folders:
```

Comment:
```suggestion
1. Depending on your operating system, save the preferences JSON file to one of the following folders:
```
proposal: err on the side of specificity.  we wouldn't say "the .cs file", we'd be specific.  there are any number of JSON files for various purposes

Reply:
I think, its fine as it is right now as well. Basically it is a JSON file with a specific name. Don't think there is a specific format for it to call it a preferences JSON file.